Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,12 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def determine_dynamic_linker_path(formula)
+    if formula && ["glibc", "glibc25"].include?(formula.name)
```

Comment:
`glibc25` is an anachronism that should be deleted. I intended using it for building portable bottles, but we never did. You can remove it from here.